src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const discount = 20;
+  const discountBig = 50;
+
+  let totalCost = days * costPerDay;
+
+  if (days >= 3) {
+    totalCost = days * costPerDay - discount;
+  }
+
+  if (days >= 7) {
+    totalCost = days * costPerDay - discountBig;
+  }
```

Comment:
```suggestion
  if (days >= 3) {
    return totalCost - discount;
  }

  if (days >= 7) {
    return totalCost  - discountBig;
  }
```

Reply:
@alexandra-protyanova  it will not work correctly. If the days will be 7 or more it will return "totalCost - discount" but not "totalCost  - discountBig" as we need

Reply:
you can replace `ifs` and it will work correct)